src/shared/mock/assets.ts

```diff
@@ -1,20 +1,20 @@
-import { Asset, AssetBNB, AssetRune67C, AssetRuneB1A } from '@xchainjs/xchain-util'
+import { Asset, AssetBNB, AssetRuneNative } from '@xchainjs/xchain-util'
 
 type Assets = 'RUNE' | 'BNB' | 'FTM' | 'TOMO' | 'BOLT'
 
 type AssetsTestnet = Record<Assets, Asset>
 type AssetsMainnet = Record<Assets, Asset>
 
 export const ASSETS_TESTNET: AssetsTestnet = {
-  RUNE: AssetRuneB1A,
+  RUNE: AssetRuneNative,
   BNB: AssetBNB,
```

Comment:
Pls remove `RUNE` + `BNB` from `ASSSETS_TESTNET` and `ASSETS_MAINNET`, we can use `AssetNativeRune` and `AssetBNB` from `xchain-util`